modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/PodcastHeaderBottom.kt

```diff
@@ -0,0 +1,116 @@
+
+private const val collapsedLines: Int = 3
+
+@Composable
+fun PodcastHeaderBottom(
+    title: String,
+    category: String,
+    description: String,
+    onDescriptionClicked: () -> Unit,
+    modifier: Modifier = Modifier,
+    ratingsContent: @Composable () -> Unit = {},
+    podcastInfoContent: @Composable () -> Unit = {},
+) {
+    var isTitleExpanded by remember { mutableStateOf(false) }
+    var isDescriptionExpanded by remember { mutableStateOf(false) }
+    val interactionSource = remember { MutableInteractionSource() }
+
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .clipToBounds()
+            .padding(bottom = 8.dp)
+            .padding(horizontal = 16.dp)
+            .background(MaterialTheme.theme.colors.primaryUi02),
+    ) {
+        TextH20(
+            text = title,
+            maxLines = if (isTitleExpanded) Int.MAX_VALUE else collapsedLines,
+            overflow = if (isTitleExpanded) TextOverflow.Clip else TextOverflow.Ellipsis,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 12.dp)
+                .animateContentSize(
+                    animationSpec = tween(
+                        durationMillis = 200,
+                        easing = FastOutSlowInEasing,
+                    ),
+                )
+                .clickable(
+                    indication = null,
+                    interactionSource = interactionSource,
+                    onClick = {
+                        isTitleExpanded = !isTitleExpanded
+                    },
+                ),
+        )
+
+        TextP40(
+            text = category,
+            fontSize = 15.sp,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 4.dp),
+            color = MaterialTheme.theme.colors.primaryText02,
+        )
+
+        ratingsContent()
+
+        Divider(
+            color = MaterialTheme.theme.colors.primaryUi05,
+            thickness = 1.dp,
+        )
+
+        TextP40(
+            text = description,
+            maxLines = if (isDescriptionExpanded) Int.MAX_VALUE else collapsedLines,
+            overflow = if (isDescriptionExpanded) TextOverflow.Clip else TextOverflow.Ellipsis,
+            lineHeight = 21.sp,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(top = 12.dp, bottom = 16.dp)
+                .animateContentSize(
+                    animationSpec = tween(
+                        durationMillis = 200,
+                        easing = FastOutSlowInEasing,
+                    ),
+                )
+                .clickable(
+                    indication = null,
+                    interactionSource = interactionSource,
+                    onClick = {
+                        isDescriptionExpanded = !isDescriptionExpanded
+                        onDescriptionClicked.invoke()
+                    },
+                ),
+        )
```

Comment:
The HTML link isn't working in the new version. But the ellipsis works better in the new version.

| Before | After |
| --- | --- |
| ![Screenshot_20250130_173635](https://github.com/user-attachments/assets/4cd7aeca-5284-4f3d-8d35-7d32bda8f6ba) | ![Screenshot_20250130_173511](https://github.com/user-attachments/assets/e38bd2d1-6272-4ea8-a094-c3f1c68f001e) |

Reply:
Nice catch! it's fixed here [4799320](https://github.com/Automattic/pocket-casts-android/pull/3513/commits/479932004262c49e2ee1a511b1d0a9ab4942c916)